.github/workflows/build.yml

```diff
@@ -31,20 +31,20 @@ env:
 
 jobs:
   base:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Any reason not to use 22.04?

Reply:
It's not the current latest stable in virtual envs afaik: https://github.com/actions/runner-images#available-images